invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/DeleteButton.js

```diff
@@ -41,6 +41,7 @@ export const DeleteButton = (props) => {
 
   useEffect(() => {
     if (modalOpen) cancelBtnRef?.current?.focus();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
is there a way to fix this?

Reply:
The only possible solution is define the cancelBtnRef variable inside the function (at least according to the doc I found). I decided against it an instead used the eslint-disable, as the variable is used in more places in the same component.